src/common/boards/nrf_PCA10184_0.1.0_54L20.json

```diff
@@ -0,0 +1,101 @@
+{
+    "board": {
+        "boardVersion": "PCA10184",
+        "boardRevision": "0.1.0",
+        "boardName": "nRF54L20 PDK (hw rev. 0.1.0)"
+    },
+    "pins": [
+        {
+            "type": "vcom",
+            "id": "vcom0",
+            "name": "VCOM0",
+            "enable": {
+                "pin": 42
+            },
+            "hwfc": {
+                "pin": 20
+            }
+        },
+        {
+            "type": "vcom",
+            "id": "vcom1",
+            "name": "VCOM1",
+            "enable": {
+                "pin": 22
+            },
+            "hwfc": {
+                "pin": 23
+            }
+        },
+        {
+            "type": "slide",
+            "id": "ledcontrol",
+            "title": "LED Power",
+            "label": "Disable LEDs to use the GPIOs for other purposes.",
+            "tooltip": "The LEDs are enabled by providing 3.3 V to LED0-LED3.",
+            "enable": {
+                "pin": 45
+            },
+            "alternatives": ["GPIO", "LED"]
+        },
+        {
+            "type": "switch",
+            "id": "swd-control",
+            "title": "Software Debugger (SWD)",
+            "label": "Disable to use an external debugger on the target chip.",
+            "tooltip": "When enabled, the SWD is connected and the IMCU is acting as the debugger. When disabled, the SWD is disconnected and you can use an external debugger connected to the Debug In header.",
+            "enable": {
+                "pin": 6
+            }
+        }
+    ],
+    "pmicPorts": [
+        {
+            "type": "voltage",
+            "port": [1, 2],
+            "portId": ["vddio", "vdd-dbgr"],
```

Comment:
what looks fragile: port and portId if arrays must have the same length and match each other. easy to have a typo

Reply:
I would not be too concerned but if you are you can do some unit test t check all pmicPorts.portId.length === pmicPorts.port.length for all JSON we have

Reply:
im concerned a bit because `port` can be just an integer and `portId` can be just a string, so mixing up their nature a bit and adding a small layer of complexity